.travis.yml

```diff
@@ -45,9 +45,9 @@ before_script:
 script:
   - mvn verify -DskipTests
   - travis_retry mvn -B clean apache-rat:check
-  - travis_retry mvn -B package jacoco:report coveralls:report
+  - travis_retry mvn -B install jacoco:report coveralls:report
```

Comment:
ITTest is designed to execute after we run successfully. We have two tasks when building, after success is a heavy operation. If we remove it, we must opt for the core ITTest case. Meanwhile, The queuing problem is very serious and will be exacerbated if you mix long-running tasks in a light task. Our CI/CD system design has been carefully considered. When optimizing the CI/CD system, we should consider all aspects.